data/migrations/20190112091830_projects.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('projects', table => {
+        table.increments();
+        table.string('project_name').notNullable().unique();
+        table.string('description').notNullable();
+        table.boolean('completed').defaultTo(false)
```

Comment:
I love to see `defaultTo`, but note that this won't operate as expected while using SQLite3! It's a great thing to have for production w/ something like Postgres, though.
Instead, since you have `useNullAsDefault: true` in your knex development config, `completed` will be null if nothing is passed in when posted.